packages/calcite-components/src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -366,6 +366,114 @@ describe("calcite-input-time-picker", () => {
       expect(await inputTimePicker.getProperty("value")).toBe("01:02");
       await assertDisplayedTime(page, "01:02 AM");
     });
+
+    describe("value", async () => {
```

Comment:
Suggestion: "value" ➡️ "direct value setting" (it would help simplify individual test names too)